roles/ci_gen_kustomize_values/templates/common/olm/values.yaml.j2

```diff
@@ -0,0 +1,2 @@
+data:
+    openstack-operator-image: {{ cifmw_ci_gen_kustomize_values_ooi_image | default('quay.io/openstack-k8s-operators/openstack-operator-index:latest', true) }}
```

Comment:
It's never going to use the default value as we are enforcing the user to provide one here: https://github.com/openstack-k8s-operators/ci-framework/pull/1239/files#diff-335b6a30d3df971b4acd157f143baba7b7659ce77fdf4ac3d1c1a14dce6271e5

Reply:
fixed :D